test/functional/driver/driver-e2e-specs.js

```diff
@@ -83,30 +83,6 @@ describe('XCUITestDriver', function () {
         simsAfter.should.equal(simsBefore);
       });
 
-      it('with udid: uses sim and shuts it down afterwards', async () => {
```

Comment:
gotcha. I'm not sure what good these assertions were ever actually doing in that case. I assumed it was asserting that the sim was shut down after the test. it looks like that may be validated by `(await simBooted(sim)).should.be.false`, so what do we get by asserting that we have the same number of sims? That we didn't accidentally create an extra one?